### ISSUE TITLE
TBC Retri seal twisting

### ISSUE THREAD

**Issue by Holmbein** [elysium-project/tbc-bug-tracker#512]
**Summary:** 
This is about seal twisting, mainly as a belf ret paladin for PvE. It's about seal twisting between Seal of Command and Seal of Blood. 

**Steps to Reproduce:** 
To seal twist, the paladin will begin with Seal of Command. Due to a short delay on Seal of Command, 0,5 seconds, it gives a small window to use Seal of Blood. Seal of Blood is a seal that deals holy damage on each melee attack. Seal of Command is counted as a melee attack. 
**Expected Results:** 
When the animation of a Seal of Command proc happens, the paladin can swap to Seal of Blood. If succeed this well happen : White hit+seal of command hit+ seal of blood hit + seal of blood hit.
**Actual Result:**
On the previous TBC beta the above didn't happen. Seal of Blood didn't proc off the melee hits (white+seal of command)
**Additional Information:**
https://www.youtube.com/watch?v=ygvWHckU6AQ
https://www.youtube.com/watch?time_continue=76&v=w5YpDd4z7gw

**Comment by ZicoQA** [comment]
This is due to the Special vendor had the wrong Seal of Blood spell to train paladins. Seal of Blood is supposed to be learnt through a quest.

**Comment by Ziggy-QA** [comment]
I had learned the correct ability [31892] Seal of Blood in my book and tested this. This is extremely hard to test for me as I generally have 185 damage. But after testing this for about an hour, I was not able to replicate seal twisting.

First - I notice that the animation is different from retail. Atleast, on a female blood elf paladin, when it procs, two animations happen.

1) The first animation is a very fast strike, and on hit applies the damage
2)Immediatly after, the normal "twirl" animation of the blood elf female happens, and damage is not applied.

It appears as though seal of blood immediately loses its aura upon switching, and does not last for 2 seconds after as described.

**Comment by Holmbein** [comment]
If I understand it right you are using Seal of blood first and tried to switch? The paladin must use Seal of command first and when a proc happens, the paladin must switch to seal of blood. Although its a very small window.

**Comment by Ziggy-QA** [comment]
> 
> 
> If I understand it right you are using Seal of blood first and tried to switch? The paladin must use Seal of command first and when a proc happens, the paladin must switch to seal of blood. Although its a very small window.

Yes. I was using Seal of command and then switched to seal of blood. Several other members of the QA team (Although admittedly, with similar ping) tried this aswell. The result was exactly as reported.
